LibVLCSharp/Shared/Media.cs

```diff
@@ -253,10 +254,10 @@ static IntPtr CtorFromInput(LibVLC libVLC, MediaInput input)
             if (input == null) throw new ArgumentNullException(nameof(input));
 
             return Native.LibVLCMediaNewCallbacks(libVLC.NativeReference,
-                LibVLCCallbacks.OpenMediaCallbackHandle,
-                LibVLCCallbacks.ReadMediaCallbackHandle,
-                LibVLCCallbacks.SeekMediaCallbackHandle,
-                LibVLCCallbacks.CloseMediaCallbackHandle,
+                OpenMediaCallbackHandle,
+                ReadMediaCallbackHandle,
+                SeekMediaCallbackHandle,
+                CloseMediaCallbackHandle,
                 GCHandle.ToIntPtr(input.ThisHandle));
```

Comment:
Can we rename "ThisHandle" to something else (GCHandle maybe?)

Reply:
Done